sdk/communication/AzureCommunicationChat/Package.swift

```diff
@@ -30,84 +30,49 @@
 import PackageDescription
 
 let package = Package(
-    name: "AzureSDK",
+    name: "AzureCommunicationChat",
     platforms: [
-        .macOS(.v10_13), .iOS(.v12)
+        .iOS(.v12)
     ],
     products: [
-        .library(name: "AzureCore", targets: ["AzureCore"]),
-        .library(name: "AzureCommunication", targets: ["AzureCommunication"]),
-        .library(name: "AzureCommunicationChat", targets: ["AzureCommunicationChat"]),
+        .library(name: "AzureCommunicationChat", targets: ["AzureCommunicationChat"])
     ],
     dependencies: [
+        .package(name: "AzureCore", url: "https://github.com/Azure/SwiftPM-AzureCore.git", .branch("master")),
+        .package(
+            name: "AzureCommunication",
+            url: "https://github.com/Azure/SwiftPM-AzureCommunication.git",
+            .branch("master")
```

Comment:
I assume at some point these will turn into a release tags over a branch reference to master.